src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -22,54 +21,28 @@
   >
     <template #pre-title="{ closeNav, isOpen, currentBreakpoint, className }" v-if="displaySidenav">
       <div :class="className">
-        <transition name="sidenav-toggle">
-          <div
-            v-show="sidenavHiddenOnLarge"
-            class="sidenav-toggle-wrapper"
+        <div class="sidenav-toggle-wrapper">
+          <button
+            :aria-label="$t('navigator.open-navigator')"
+            :id="baseNavOpenSidenavButtonId"
+            class="sidenav-toggle"
+            :tabindex="isOpen ? -1 : null"
+            @click.prevent="handleSidenavToggle(closeNav, currentBreakpoint)"
           >
-            <button
-              :aria-label="$t('navigator.open-navigator')"
-              :id="baseNavOpenSidenavButtonId"
-              class="sidenav-toggle"
-              :tabindex="isOpen ? -1 : null"
-              @click.prevent="handleSidenavToggle(closeNav, currentBreakpoint)"
-            >
-            <span class="sidenav-icon-wrapper">
-              <SidenavIcon class="icon-inline sidenav-icon" />
-            </span>
-            </button>
-            <span class="sidenav-toggle__separator" />
-          </div>
-        </transition>
+          <span class="sidenav-icon-wrapper">
+            <SidenavIcon class="icon-inline sidenav-icon" />
+          </span>
+          </button>
+        </div>
       </div>
     </template>
     <template #default>
-      <slot
-        name="title"
-        v-bind="{ rootLink, linkClass: 'nav-title-link', inactiveClass: 'inactive' }"
-      >
-        <router-link
-          v-if="rootLink"
-          :to="rootLink"
-          class="nav-title-link"
-        >
-          {{ $t('documentation.title') }}
-        </router-link>
-        <span v-else class="nav-title-link inactive">{{ $t('documentation.title') }}</span>
-      </slot>
+      <slot name="title" />
     </template>
     <template #tray="{ closeNav }">
-      <Hierarchy
-        :currentTopicTitle="title"
-        :isSymbolDeprecated="isSymbolDeprecated"
-        :isSymbolBeta="isSymbolBeta"
-        :parentTopicIdentifiers="hierarchyItems"
-        :currentTopicTags="currentTopicTags"
-        :references="references"
-      />
       <NavMenuItems
         class="nav-menu-settings"
-        :previousSiblingChildren="breadcrumbCount"
+        :previousSiblingChildren="1"
```

Comment:
You are right, initially I set it to 1 because `breadcrumbCount` had a `+1` on it. https://github.com/apple/swift-docc-render/pull/772/files#diff-faae97732aa108c4a3c75b5f33955698c951ad49583851cc821ab20369fac642L167 but now I realised that it was because we were rendering "documentation" breadcrumb before the hierarchyItems.